fluid/fluid.cpp

```diff
@@ -38,6 +38,11 @@ namespace FluidS {
 
 bool Fluid::initialized = false;
 
+/* better than a macro to determine inappropriate values for notes*/
+bool validNote(const int input) {
+      return (input < 255 && input > -1);
```

Comment:
See discussion at https://github.com/musescore/MuseScore/pull/5853#pullrequestreview-378824367

Reply:
How about this